c2corg_ui/static/js/documentediting.js

```diff
@@ -221,7 +231,7 @@ app.DocumentEditingController.prototype.errorRead_ = function(response) {
  */
 app.DocumentEditingController.prototype.submitForm = function(isValid) {
   if (!isValid) {
-    // TODO: use a common messaging tool to display the error
+    // Should not happen: form validity is tested before being able to submit.
```

Comment:
Indeed. But we never know: perhaps the validity check has been omitted in the editing form.
I think I will use the standard alerts service instead of a raw alert(), using the regular way to give feedbacks to the user.